doc/orchestrator.rst

```diff
@@ -63,108 +326,207 @@ on that compute host.
 
 |colo-orc|
 
-This deployment is designed for highly performant online inference scenarios where
+Colocated deployment is designed for highly performant online inference scenarios where
 a distributed process (likely MPI processes) are performing inference with
-data local to each process.
+data local to each process. Data produced by these processes and stored in the colocated database
+can be transferred via a SmartRedis client to a standard database to become available for consumption
```

Comment:
I don't think "standard database" is really how we want to call it. Maybe "standalone"?